posthog/hogql_queries/experiments/funnels_statistics_v2.py

```diff
@@ -21,28 +21,38 @@ def calculate_probabilities_v2(
     for funnel conversion rates.
 
     This function computes the probability that each variant is the best (i.e., has the highest
-    conversion rate) compared to all other variants, including the control. It uses samples
-    drawn from the posterior Beta distributions of each variant's conversion rate.
+    conversion rate) compared to all other variants, including the control. It uses Beta
+    distributions to model conversion rates, which is the conjugate prior for binomial data.
```

Comment:
What's a "conjugate prior for binomial data"? In the code comments I'd aim for the same level of simplicity as in our docs :)